source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,265 @@
+choco install python
+```
+```shell
+brew install python3
+```
+
+### Windows
+
+**If you're already familiar with [Chocolatey][14], you can follow the commands shown to the right instead of installing manually.**
+
+<aside class="notice">
+Please remember to run choco commands in administrator powershell/command prompt.
+</aside>
+
+### Mac
+
+**If you're already familiar with [Homebrew][6], you can follow the commands shown to the right instead of installing manually.**
+
+* **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run `git` from your command prompt.
+* **Python**: Run the install command on the tab to the right to install the brew python version of your choice.
+
+## Install with Anaconda
+
+```
+conda install git
+```
+
+If you *do not* have Anaconda installed on your system, please skip this section.
+
+conda is a tool for installing and managing Python and R dependencies. Anaconda includes Python and pip, so all you need to do is install git if you haven't already. For more information, refer to the [Anaconda website][10].
+
+# Setup the Bonsai CLI
+
+> Python 2 and Anaconda
+
+```
+pip install bonsai-cli
+bonsai configure
+```
+
+> Python 3
+
+```powershell
+# same as above
+```
+```shell
+pip3 install bonsai-cli
+bonsai configure
+```
+
+From your command prompt, install the Bonsai Command Line Interface tool and configure it. `bonsai configure` will ask for your Access Key. You can find your Access Key in your [Bonsai Account Settings][8].
+
+## Clone a sample project
+
+Clone a sample simulation project with git. For this guide we'll be walking you through [OpenAI Gym's][4] Mountain Car environment, one of the supported Bonsai simulators.
+
+```
+git clone https://github.com/BonsaiAI/gym-mountaincar-sample
+```
+
+## Install requirements
+
+> Python 2 and Anaconda
+
+```
+cd gym-mountaincar-sample
+pip install -r requirements.txt
+```
+> Python 3
+
+```powershell
+# same as above
+```
+```shell
+cd gym-mountaincar-sample
+pip3 install -r requirements.txt
+```
+
+Enter into the Mountain Car folder and then install the Mountain Car requirements from OpenAI Gym.
+
+# Create your BRAIN
+
+Create your BRAIN and give it a name. You can also create a BRAIN from your [beta.bons.ai][5] BRAIN Dashboard instead of using the command line. Either way, you will view your BRAIN's progress on the BRAIN Details page.
+
+### Create from Web
+
+![BRAINS Dashboard Page][15]
+
+The dashboard has a NEW BRAIN button, pictured above. Click on it to start the BRAIN creation process, where you're provided a form to give your brain a name and description. Name can contain alpha, numerals, or hyphens; the description is arbitrary and should contain human-readable text that explains your intent for the BRAIN.
+
+### Create from Command Line
+
+```
+bonsai create myMountainCarBrain
+```
+You may give your brain a name with `bonsai create brainName` but there is no option for a description with this method.
```

Comment:
It might make sense to say "after you create your BRAIN, you can change it's description from the web on the BRAIN Details page" or something like that.